### PR TITLE
Prepare 2.53.rc.1

### PR DESCRIPTION
**Pull request by krajorama**
The release-2.53 branch already includes #14273 fix

### CODE REVIEW

**Comment by krajorama** [comment]
Force pushed to update DCO with signature in revert commit

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,13 +2,17 @@
 
 ## unreleased
 
+## 2.53.0-rc.1 / 2024-06-10
+
+* [BUGFIX] Rules: Fix Group.Equals() to take in account the new queryOffset too. Followup to #14061. #14273
```
Comment (ArthurSens) [inline CHANGELOG.md:7]: Not sure if mentioning method names will help users understand what is being fixed here. With that said, I'm struggling to find a better changelog entry 😬

Comment (krajorama) [reply CHANGELOG.md:7]: it will be removed anyway since it's fixing something introduced in the rc.0

**Comment by roidelapluie** [comment]
I'd wait for #14285

**Review by SuperQ — APPROVED** [review]

**Pull request merged**